contributors/design-proposals/storage/local-storage-pv.md

```diff
@@ -761,3 +761,124 @@ The central manager will poll each worker for their discovered volumes and creat
 them.  When a PV is released, then it will send the cleanup request to the worker.
 
 Detailed design TBD
+
+
+### Phase 3: 1.11 alpha
+
+#### Dynamic provisioning
+
+The local volume provisioner will be enhanced to support dynamic provisioning of local storage,
+which depends on the newly introduced changes in [Volume Topology-aware Scheduling](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-topology-scheduling.md),
+that is:
+
+1. When feature `DynamicProvisioningScheduling` enabled, scheduler would verify a pod's volume requirement,
+and set `annScheduledNode` on PVCs that need provisioning.
+2. New fields would be added in `StorageClass` to allow storage providers to report
+capacity limits per topology domain.
+
+The local volume provisioner need to be updated in several aspects to support the scenario.
```

Comment:
nit: /need/needs/
nit: /aspects/areas/
nit: /the scenario/dynamic provisioning/